Python/omevv/v1/scripts/GetOMIVVProfileDetails.py

```diff
@@ -0,0 +1,213 @@
+import argparse
+import sys
+import csv
+import time
+import warnings
+from webbrowser import get
+from model import consoleDecoder,repoProfileDecoder
+
+import requests,json,base64
+
+class RepoDetails:
+    def __init__(self,omevv_ip,omevv_username,omevv_pwd,domain):
```

Comment:
As this script is retrieving from legacy appliance, rename the variables to omivv